app/gui/src/project-view/components/shared/AgGridTableView.vue

```diff
@@ -142,6 +142,9 @@ useAutoBlur(() => grid.value?.$el)
 
 function onGridReady(event: GridReadyEvent<TData>) {
   gridApi.value = event.api
+  if (rowModelType.value === 'serverSide') {
+    gridApi.value.retryServerSideLoads()
+  }
```

Comment:
I wonder why this is needed? Shouldn't AgGrid properly load data as part of its initialization?

I think a short comment here would be helpful.